ui/src/app/services/sign-in.service.ts

```diff
@@ -0,0 +1,113 @@
+import 'rxjs/Rx';
+import {Injectable, NgZone} from '@angular/core';
+import {Observable} from 'rxjs/Observable';
+import {BehaviorSubject} from 'rxjs/BehaviorSubject';
+import {User} from 'app/models/user';
```

Comment:
Is it useful to keep around the models.User, or should we just use UserInfoPlus? I guess interposing our own model may provide useful separation from gapi.